.github/workflows/build.yml

```
@@ -25,28 +29,32 @@ jobs:
         with:
           path: '**/node_modules'
           key: node-modules-${{ hashFiles('**/package-lock.json') }}
-      - name: Cache Cypress
-        id: cache-cypress
-        uses: actions/cache@v3
-        with:
-          path: ~/.cache/Cypress
-          key: cypress-cache
       - name: Cache eslint
+        if: ${{ matrix.task == 'lint' }}
         id: cache-eslint
         uses: actions/cache@v3
         with:
           path: .eslintcache
           key: eslint-${{github.sha}}
```

A restore-key can be added here to have an incremental cache with `eslint`.
If a cache for the commit `sha` doesn't exist, it will retrieve the cache of the last commit.
When the lint has finished, it will save a new cache for the current commit.

```suggestion
          key: eslint-${{github.sha}}
          restore-keys: eslint-
```